.buildkite/test-pipeline.yaml

```
@@ -923,7 +923,28 @@ steps:
     - pytest -v -s tests/compile/test_silu_mul_quant_fusion.py
     # this runner has 2 GPUs available even though num_gpus=2 is not set
     - pytest -v -s tests/compile/test_fusion_all_reduce.py
-    - pytest -v -s tests/compile/test_fusions_e2e.py
+
```

This test should run a subset of the e2e fusion tests:
```suggestion
    pytest -v -s tests/compile/test_fusions_e2e.py::test_tp2_attn_quant_allreduce_rmsnorm -k "True and Llama-3.1 and -quant_fp8 and -rms_norm"
```